### PR TITLE
Remove `html_enc` from few IA Goodies

### PR DESCRIPTION
**Pull request by ManrajGrover**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`{More than 1}: Fix {Removes html_enc}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Remove `html_enc` from few IA Goodies

###### Which issues (if any) does this fix?

#3563

###### People to notify (@mention interested parties)

@moollaza 

---

Instant Answer Page:

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention
None

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/EmailValidator.pm [hunk 1]
```diff
@@ -45,8 +45,8 @@ handle remainder => sub {
 
     return $message, structured_answer => {
         data => {
-            title => html_enc($message),
-            subtitle => 'Email address validation: '.html_enc($address)
+            title => $message,
+            subtitle => 'Email address validation: '.$address
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/EmailValidator.pm]: Want to switch this to `Email address validation: $address"` while you're at it?

And any others now enabled too :smile:

**Comment by ManrajGrover** [comment]
@GuiltyDolphin The build stopped midway. Could you please look into it? :smile:

**Comment by GuiltyDolphin** [comment]
@ManrajGrover Still an issue? Build seems to have finished from what I can tell.

**Comment by ManrajGrover** [comment]
@GuiltyDolphin Nope, looks like someone else restarted it.

**Comment by ManrajGrover** [comment]
@GuiltyDolphin I guess we can merge this

File: lib/DDG/Goodie/MacAddress.pm [hunk 2]
```diff
@@ -43,7 +43,7 @@ handle remainder => sub {
     return unless $info;
     my (@vendor) = split(/\\n/, $info, 2);
 
-    my ($name, $addr) = map { html_enc($_); } @vendor;
+    my ($name, $addr) = map { $_; } @vendor;
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/MacAddress.pm]: A.k.a `@vendor` (mapping the identity is the same as the list).

Comment (ManrajGrover) [inline lib/DDG/Goodie/MacAddress.pm]: @GuiltyDolphin Fixed :)

**Comment by GuiltyDolphin** [comment]
@ManrajGrover Thanks!

**Pull request merged**